units/src/fee.rs

```diff
@@ -19,7 +19,7 @@ impl Amount {
     /// Checked weight ceiling division.
     ///
     /// Be aware that integer division loses the remainder if no exact division
-    /// can be made. This method rounds up ensuring the transaction fee-rate is
+    /// can be made. This method rounds up ensuring the transaction `fee_rate` is
```

Comment:
This one is not quite right, at least to how I personally write function docs. I typically only use 'code' (i.e. backticks) for identifiers, specifically function args. In this case I think you are correct in that fee-rate is wrong (I couldn't find a first page mention when web searching). Perhaps just plain 'fee rate' is better?

Reply:
I agree and have changed it.  I think the example directly below it using `fee_rate` threw me.